src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Categories = () => (
+  <section className="categories">
+    <h2>Work in  progress</h2>
+  </section>
+);
+
+export default Categories;
```

Comment:
- [ ] You have created the categories folder and categories.js file, but it is empty file, I suggest you to implement `work in progress` in that file not in the component. 

![category](https://user-images.githubusercontent.com/55635977/131731467-5a42bf6d-1c99-474a-87d0-f3d25e838fa6.png)

Reply:
Well done